proxy/src/registry.rs

```diff
@@ -177,10 +175,8 @@ impl Registry {
         fee: Balance,
     ) -> Result<Transaction, error::Error> {
         // Verify that inputs are valid.
-        let project_name = ProjectName::try_from(name.clone())
-            .map_err(|_| error::ProjectValidation::NameTooLong)?;
-        let org_id =
-            OrgId::try_from(org_id.clone()).map_err(|_| error::ProjectValidation::OrgTooLong)?;
+        let project_name = ProjectName::try_from(name.clone())?;
+        let org_id = OrgId::try_from(org_id.clone())?;
```

Comment:
Don't you need to map_err to go from the registry error to the proxy upstream one?

Reply:
We satisfy `try` semantics in our error module where we have the infrastructure to lift protocol errors into upstream domain:

https://github.com/radicle-dev/radicle-upstream/blob/1755dee156c15f85ea183d996a27841900ee5337/proxy/src/error.rs#L140-L144

Reply:
Ah, sweet. I figured that had to be done explicitly here.